addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -223,6 +223,18 @@ def in_eu(partner):
         pdf = base64.b64encode(pdf)
         pdf_name = re.sub(r'\W+', '', self.name) + '.pdf'
 
+        # tax map for 0% taxes which have no tax_line_id
+        tax_map = dict()
+        for line in self.line_ids:
+            # we have at most 1 tax_id per line
+            tax = line.tax_ids
+            if tax.l10n_it_has_exoneration:
+                tax_amount = tax.compute_all(line.price_unit)
```

Comment:
@agr-odoo How can this work?  You are not calculating the price for one piece, but for all...

Reply:
better to use the params of compute_all as intended

Reply:
ok, done :+1: